<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab {height:70px; width:160px;}</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabHome">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>791</width>
        <height>522</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTools">
     <attribute name="title">
      <string>Tools</string>
     </attribute>
     <widget class="QToolBox" name="toolsTab">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>791</width>
        <height>522</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>791</width>
         <height>441</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Character Tools</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Map Tools</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>791</width>
         <height>441</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Community Tools</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCommunity">
     <attribute name="title">
      <string>Network</string>
     </attribute>
     <widget class="QGroupBox" name="groupButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Choose connection</string>
      </property>
      <widget class="QPushButton" name="launchServerButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Launch server</string>
       </property>
      </widget>
      <widget class="QPushButton" name="connectionButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Connection to server</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupServer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>511</height>
       </rect>
      </property>
      <property name="title">
       <string>Server mod</string>
      </property>
      <widget class="QTextEdit" name="serverText">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>741</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="stopServerButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>450</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop serveur</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupClient">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>511</height>
       </rect>
      </property>
      <property name="title">
       <string>Client mod</string>
      </property>
      <widget class="QLineEdit" name="fileLine">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>431</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="clientExitButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>450</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Client exit</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="clientText">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>741</width>
         <height>361</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="sendFile">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>30</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Send file</string>
       </property>
      </widget>
      <widget class="QPushButton" name="getFile">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Get file</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSearch">
     <attribute name="title">
      <string>Search</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabSettings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
